### ISSUE TITLE
Espressi-IDE 3.0.0 doesn't have "Open declaration" in context menu

### ISSUE THREAD

**Issue by dariomingarelli** [espressif/idf-eclipse-plugin#995]
Espressi-IDE 3.0.0 doesn't have "Open declaration" in context menu.
I try also "F3" but doesn't work.

To reproduce:
A) highlight any var and click with mouse "right button" (two finger click on touchpad for MacOs) and search for "Open declaration".
B) highlight any var and press "F3"

**Comment by zjanosy** [comment]
Use Ctrl-click instead.

**Comment by dariomingarelli** [comment]
> Ctrl-click on the variable or function name to open the declaration.

That works (CMD + click on MacOs).

By the way... I think that is useful also to add it on context menu!

**Comment by zjanosy** [comment]
I agree, although once you get used to the Ctrl-click method, it may be even more convenient. But I also miss opening a header file using the context menu, and I did not find how to do this in the new editor.

**Comment by zjanosy** [comment]
Actually, Ctrl-click works for opening a header as well...

**Comment by thaivuongha** [comment]
Me too. ESP IDE 3.0.0, I pressed F3 or Ctrl+ClickLeft but can not open declaration. All are hidden

**Comment by kolipakakondal** [comment]
Hi @thaivuongha Could you please check the below key configured or not? C/C++ Editor in 3.0.0 is LSP based which uses Generic Code Editor keys.
 
<img width="1153" alt="Screenshot 2024-08-09 at 5 49 28 AM" src="https://github.com/user-attachments/assets/95cd959f-aceb-4ae3-bf25-2d9b86b46232">

**Comment by thaivuongha** [comment]
Hi @kolipakakondal . I tried and succeeded. Thank you so much!

**Comment by kolipakakondal** [comment]
Closing this issue as it has been addressed. With CDT-LSP 3.0.0, we also have the "Open Declaration" context menu available.

For Espressif-IDE users, this will be included in the upcoming v3.4.0 release.

<img width="395" alt="Image" src="https://github.com/user-attachments/assets/1fb5d022-8823-47b7-bc0b-8dff378b71c1" />
